index.html

```diff
@@ -1,12 +1,27 @@
 <!DOCTYPE html>
-  <html lang="en">
+  <html lang="en" data-ng-app="todo">
   <head>
      <meta charset="UTF-8">
      <title>Angular ToDo App</title>
+     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+
      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.2/angular.min.js"></script>
-     <script src="app.js"></script>
+     <script type="text/javascript" src="app.js"></script>
+
   </head>
   <body>
-    <h1>To Do!</h1>
+      <div data-ng-controller="todo_controller as vm">
+          <h1>To Do!</h1>
+          <input type="text" data-ng-model="vm.new_todo.task">
+          <button ng-click="vm.create()">Create</button>
+
+          <div ng-repeat="item in vm.data">
+                <p contentEditable="true">{{item.task}}</p>
+                <input type="text" data-ng-model="item.edit_todo.task">
```

Comment:
What exactly is `edit_todo`?

If you're trying to access a task that belongs to a particular `item`, then you can do that using `item.task`.

If you're trying to create a placeholder task that will ultimately be added to data after you're done editing, you would store that in something like `vm.edit_todo`.

Happy to talk about this further if you'd like.